app/webpacker/components/RegistrationsV2/Register/StripeWrapper.jsx

```diff
@@ -44,8 +44,7 @@ export default function StripeWrapper({
 
   const handleDonation = useCallback(async (newDonationAmount) => {
```

Comment:
WIth the recent change abandoning `refetch`, you don't need `async` anymore. You also don't need this callback anymore, as you can just pass around `setDonationAmount` directly.